web3/inpage.js

```diff
@@ -59,13 +59,13 @@ class Web3FrameProvider {
       }
 
       if (ev.data.err === TRANSPORT_UNINITIALIZED) {
-        if (this.isConnected) {
+        if (this.isConnected()) {
```

Comment:
It has to be a function, it's part of the interface for Provider in web3 library.